concent_api/api-integration-multi-threads-test.py

```diff
@@ -0,0 +1,105 @@
+#!/usr/bin/env python3
+
+import hashlib
+import os
+import sys
+import time
+from datetime import datetime
+from threading import Thread
+
+import requests
+from golem_messages.shortcuts import dump
+
+from api_testing_common import api_request
+from api_testing_common import assert_condition
+from api_testing_common import count_fails
+from api_testing_common import get_force_get_task_result
+from api_testing_common import REQUESTOR_PRIVATE_KEY
+from api_testing_common import run_tests
+from common.helpers import get_current_utc_timestamp
+
+os.environ.setdefault("DJANGO_SETTINGS_MODULE", "concent_api.settings")
+
+now = datetime.now()
+testing_subtask_id = str(now.year) + str(now.month) + str(now.day) + str(now.hour) + str(now.minute) + str(now.second) + str(now.microsecond)
+
+responses = []  # type: list
+
+number_of_testing_threads = 3
+
+
+def multi_threads(number_of_threads: int):
+    def call_function_in_threads(func):
+        def wrapper(*args, **kwargs):
+            for i in range(number_of_threads):
+                t = Thread(target=func, args=(*args,), kwargs={**kwargs})
+                t.start()
+
+        return wrapper
+    return call_function_in_threads
+
+
+@count_fails
+def test_case_that_multiple_requests_by_one_subtask_will_not_be_cause_of_server_error(cluster_consts, cluster_url, test_id):
+
+    send_correct_request(cluster_consts=cluster_consts, cluster_url=cluster_url)
+
+    for i in range(3 * number_of_testing_threads):
```

Comment:
Wait, why it is `3 *` ? Wasn't it just `3` previously ?

Reply:
It were hardcoded 3 threads. In this case it is really number of repeats --> waiting time. I decided to do it like this, because if someone would change `number_of_testing_threads` possible waiting time should be also different. And it should depend on number of testing threads